src/presentation/mesh-table-factory.ts

```diff
@@ -9,7 +10,8 @@ export class MeshTableFactory {
   }
 
   buildMeshTable(): MeshTable {
-    if (this.isatty) {
+    if (this.isatty && os.default.platform() !== "windows") {
```

Comment:
d: we mix Deno.build.os !== "windows" and os.default.platform() !== "windows" maybe really consider this central variable at some meaningful spot, maybe in one of the init.ts files?

Reply:
makes total sense, will do